src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -14,21 +22,93 @@ export const ARShowScreen = ({ navigation, route }) => {
   }, []);
 
   const parser = async () => {
-    await objectParser({ item: data[index], setObject });
+    // TODO: the `onPress` prop is just for testing
+    await objectParser({ item: data[index], setObject, onPress: () => navigation.goBack() });
+
     setIsLoading(false);
   };
 
-  if (isLoading || !object) return <LoadingSpinner loading />;
+  if (isLoading || !object || !object.vrx) return <LoadingSpinner loading />;
```

Comment:
if the object is not a `vrx` we show a loading spinner, ok. but for how long? will there change anything? or should we handle this case separately somehow?

Reply:
This is just to prevent the application from crashing during the testing process. Already in case of such a thing, an alert is shown on the screen.